src/WP_Error/HttpError.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * HTTP errors.
+ *
+ * @package SatisPress
+ * @license GPL-2.0-or-later
+ * @since 0.3.0
+ */
+
+declare ( strict_types = 1 );
+
+namespace SatisPress\WP_Error;
+
+use WP_Error;
+use WP_Http as HTTP;
+
+/**
+ * HTTP errors class.
+ *
+ * @since 0.3.0
+ */
+class HTTPError extends WP_Error {
```

Comment:
Any issue with naming here for PSR-4? (Like the Ajax class)

Reply:
I took a look after renaming the other one and didn't see any issues. Here's what it says about case:

> Alphabetic characters in the fully qualified class name MAY be any
combination of lower case and upper case.

Reply:
I noticed, but think the new naming looks a little nicer. PSR-4 addresses case-sensitivity as well:

> All class names MUST be referenced in a case-sensitive fashion.